### PR TITLE
Alerts for deck name checking

### PR DESCRIPTION
**Pull request by Tyler-Brenneman**
Checks to make sure a new deck name is a valid length, and displays an alert if it isn't.

Let me know if you think the alert wording should be different.

### CODE REVIEW

File: Cue/app/tabs/library/LibraryHome.js [hunk 1]
```diff
@@ -111,7 +111,16 @@ class LibraryHome extends React.Component {
       [
         {text: 'Cancel', style: 'cancel'},
         {text: 'Create', onPress: (deckName) => {
-          if (deckName && deckName.length) {
+          if (!deckName || !deckName.length) {
+            Alert.alert(
+              Platform.OS === 'android' ? 'Enter a deck name' : 'Enter a Deck Name',
```
Comment (mattyu007) [inline Cue/app/tabs/library/LibraryHome.js]: -----
**Deck Name Can’t Be Empty**

A good deck name is short but descriptive.

-----
? I'm not sure a second sentence is really necessary though. Need a second opinion.

File: Cue/app/tabs/library/LibraryHome.js [hunk 2]
```diff
@@ -111,7 +111,16 @@ class LibraryHome extends React.Component {
...
+            )
+          } else if (deckName.length > 255) {
+            Alert.alert(
+              Platform.OS === 'android' ? 'Deck name too long' : 'Deck Name Too Long',
+              'Please enter a shorter deck name',
```
Comment (mattyu007) [inline Cue/app/tabs/library/LibraryHome.js]: -----
**Deck Name Too Long**

Use a shorter name for your deck.

-----
? I don't feel too strongly about either, but I'd avoid using the word "please".

**Review by mattyu007 — COMMENTED** [review]
Ugh, I wish we could just disable the Create button instead of needing to use more alerts.

Question: Should we re-display the alert if a validation error occurs?

File: Cue/app/tabs/library/LibraryHome.js (hunk 1 above)
Comment (ClaytonPassmore) [reply Cue/app/tabs/library/LibraryHome.js]: Yeah I feel like the title gets the message across

File: Cue/app/tabs/library/LibraryHome.js (hunk 2 above)
Comment (ClaytonPassmore) [reply Cue/app/tabs/library/LibraryHome.js]: pls

**Review by mattyu007 — APPROVED** [review]

**Pull request merged**